go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/coredns/coredns
 
-go 1.21
+go 1.22.5
```

Comment:
We would like to support the latest two Go releases.

Reply:
> go: k8s.io/api@v0.30.3 requires go >= 1.22.0 (running go 1.21.0; GOTOOLCHAIN=go1.21.0)

This is required by v0.30.3. 

Or we may wait for golang 1.23 release and then bump.

Reply:
Yes, we will need to hold back on k8s.io dependencies until 1.23.